meta/saiserialize.cpp

```diff
@@ -2111,7 +2111,7 @@ void sai_deserialize_pointer(
 {
     SWSS_LOG_ENTER();
 
-    sai_deserialize_number(s, (uint64_t&)ptr, true);
+    sai_deserialize_number(s, (uintptr_t &)ptr, true);
```

Comment:
also why this is changed ?

Reply:
same cast-align warning, cannot convert a pointer to a 64 bit as size is increased.